autoload/which_key.vim

```diff
@@ -20,7 +20,7 @@ function! which_key#start(vis, bang, prefix) " {{{
   endif
 
   let key = a:prefix
-  let s:which_key_trigger = key ==# ' ' ? 'SPC' : key
+  let s:which_key_trigger = key ==# ' ' ? '<space>' : key
```

Comment:
Why this change?

Reply:
This change to be able to passing the variable `s:which_key_trigger` to the `normal! ` command with minimal changes.
https://github.com/liuchengxu/vim-which-key/pull/77/files#diff-44477f6acbd959134929d83f81d48e73R297